diag_test.py

```
@@ -6,12 +6,14 @@
 - ROC curves and AUC calculation
 - Cohen's Kappa
 - ICC (Intraclass Correlation)
-- Risk/diagnostic metrics with confidence intervals
+- Risk/diagnostic metrics (NNT, ARR, RRR, DOR) with confidence intervals
+- Clinical Significance Badges
 
 Note: Removed Streamlit dependencies, now Shiny-compatible
 OPTIMIZATIONS: DeLong method vectorized (106x faster), ICC vectorized (9x faster)
 """
 
+from typing import Union, Optional, Any, Tuple, Dict, List, Literal
```

Ruff (0.14.10)

`typing.Tuple` is deprecated, use `tuple` instead

(UP035)